services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: str = None,
```

Comment:
First of all, it accepts list of ints. The same with actors_ids
Also it will be optional because accepts None(fix this case for all code)